.travis.yml

```diff
@@ -29,7 +33,14 @@ matrix:
       - python: "3.3"
         env: DJANGO=1.10
       - python: "3.3"
+        env: DJANGO=1.11
```

Comment:
For now, that seems correct? There are more special cases to include than special cases to exclude.